lib/sinon/default-behaviors.js

```diff
@@ -181,6 +183,12 @@ module.exports = {
         return fake;
     },
 
+    resolvesThis: function resolvesThis(fake) {
+        fake.resolveThis = true;
+        fake.resolve = false;
+        fake.reject = false;
```

Comment:
Okay now resolves, rejects and resolvesThis all act on the same flags.
I set returnValue to undefined as the stub resolves with itself.
Is it okay by you?